playbooks/roles/ocp-config/tasks/extract.yaml

```diff
@@ -31,6 +31,41 @@
   args:
     chdir: "{{ tools_dir }}"
 
+- name: Extract OCP4 tools from release image ( local-registry )
+  when: enable_local_registry
+  shell: |
+    oc adm release extract --tools {{ release_image_override }} --registry-config='{{ ansible_env.HOME }}/.openshift/pull-secret-updated'
+  args:
+    chdir: "{{ tools_dir }}"
+
+- name: Check if FIPS is enabled
+  ansible.builtin.command: grep -q 1 /proc/sys/crypto/fips_enabled
+  register: fips_check
+  changed_when: false
+  failed_when: false
+
+- name: Check if openshift-install binary exists
+  ansible.builtin.stat:
+    path: "/usr/local/bin/openshift-install"
+  register: binary_check
```

Comment:
So this is changed only if "/usr/local/bin/openshift-install" file exists?
This could be `which openshift-install` and replace that.